tests/unit/util/json_util_test.py

```diff
@@ -233,41 +233,40 @@ def test_non_ascii_data_non_utf_locale_coding_orjson(self):
 
 @pytest.mark.json_lib
 class TestDefaultJsonLibrary(ScalyrTestCase):
+    def setUp(self):
+        super(TestDefaultJsonLibrary, self).setUp()
+        # Save original modules so we can restore them after tests.
+        self._original_modules = {}
+        for name in ["orjson", "ujson", "json"]:
```

Comment:
In the future if we assume frozen binaries with Python 3 everyone we could just support ``orjson`` to simplify things.

Reply:
Agree, and standard `json` lib as a fallback.